src/plone/restapi/services/site/get.py

```diff
@@ -54,6 +59,11 @@ def __call__(self, expand=False):
             }
         )
 
+        if HAS_MULTILINGUAL and IPloneAppMultilingualInstalled.providedBy(self.request):
+            result["site"]["plone.is_multilingual"] = True
```

Comment:
@davisagli should we use the prefix in this one? I assume we left the others because they are mapped like this to the registry values.